doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,337 @@
+            "secretAlias"
+          ]
+        }
+      ]
+    }
+  },
+  "supportedArchitectures": {
+    "type": "array",
+    "description": "The CPU architectures that this connector image was built for. At least one must be specified.",
+    "items": [
+      {
+        "type": "string",
+        "description": "One of the CPU architectures this connector image was built for. For example: 'linux/amd64', 'linux/arm64', or 'windows/amd64'"
+      }
+    ],
+    "minItems": 1,
+    "uniqueItems": true
+  },
+  "sourceCode": {
+    "type": "object",
+    "description": "Details about the source code that compiled this connector",
+    "properties": {
+      "language": {
+        "type": "string",
+        "description": "The programming language used to write this connector"
+      },
+      "languageVersion": {
+        "type": "string",
+        "description": "The version of the programming language used to write this connector"
+      },
+      "sdks": {
+        "type": "object",
+        "description": "Details about the versions of the AIO SDKs used to write this connector (if they were used)",
+        "properties": {
+          "mqttPackageVersion": {
+            "type": "string",
+            "description": "The version of the Azure IoT Operations MQTT package"
+          },
+          "protocolPackageVersion": {
+            "type": "string",
+            "description": "The version of the Azure IoT Operations protocol package"
+          },
+          "servicesPackageVersion": {
+            "type": "string",
+            "description": "The version of the Azure IoT Operations services package"
+          },
+          "connectorPackageVersion": {
+            "type": "string",
+            "description": "The version of the Azure IoT Operations connector package"
+          }
+        }
+      }
+    },
+    "required": [
+      "language",
+      "languageVersion"
+    ]
+  },
+  "aioMetadata": {
+    "type": "object",
+    "description": "The AIO requirements to run this connector",
+    "properties": {
+      "aioMinVersion": {
+        "type": "string",
+        "description": "The minimal (inclusive) version that can run this connector."
+      },
+      "aioMaxVersion": {
+        "type": "string",
+        "description": "The maximal (inclusive) version that can run this connector."
+      }
+    }
+  },
+  "additionalConfigurationSchema": {
+    "description": "The connector-level JSON schemas for the 'additional configuration' field. A JSON schema must be only if that field is used by this connector.",
+    "$ref": "#/definitions/schema"
+  },
+  "inboundEndpoints": {
+    "type": "array",
+    "description": "The array of inbound endpoints in a device that this connector will be deployed for.",
+    "items": [
+      {
+        "type": "object",
+        "properties": {
+          "endpointType": {
+            "type": "string",
+            "description": "The type of this inbound endpoint."
+          },
```

Comment:
The description should be:
The type of this inbound endpoint including the version like "Microsoft.Media:1.2.0"